apps/nameguard.io/src/components/organisms/ContactUsForm.tsx

```diff
@@ -274,30 +253,19 @@ export const ContactUsForm = ({ title }: ContactUsFormProps) => {
               >
                 Telegram (optional)
               </label>
-              <div className="relative">
-                <span className="text-sm leading-5 font-medium text-gray-500 absolute left-2 top-2.5">
-                  @
-                </span>
-                <input
-                  type="text"
-                  id="telegram"
-                  autoComplete="off"
-                  disabled={isLoading}
-                  onChange={handleInputChange}
-                  name={FormFields.Telegram}
-                  className={cc([
-                    "block w-full rounded-md pr-3.5 pl-6 py-2 text-gray-900 shadow-sm border border-gray-300 placeholder:text-gray-400 focus:border-gray-600 hover:border-gray-400 outline-none sm:text-sm sm:leading-6",
-                    {
-                      "border-red-300": validationErrors[FormFields.Telegram],
-                    },
-                  ])}
-                />
-                {validationErrors[FormFields.Telegram] && (
-                  <span className="mt-2 text-sm font-normal text-red-600">
-                    {validationErrors[FormFields.Telegram]}
-                  </span>
-                )}
-              </div>
+              <Input
```

Comment:
@eduramme Hey can you please investigate the alignments here? We might need your help to make some adjustments to the `Input` component to fix this.

Here's a screenshot from this form field as it currently exists on nameguard.io:

![CleanShot 2024-09-11 at 14 59 35](https://github.com/user-attachments/assets/dea6579c-8992-4274-b82b-089286ef558e)

As you can see, the vertical alignment is centered and consistent for both the slot value "@" and the value being typed into the field "abc". Everything here looks great.

However, here's a screenshot of the current preview environment:

![CleanShot 2024-09-11 at 15 03 01](https://github.com/user-attachments/assets/230ecc25-db10-41aa-92fe-fa3e1f90667a)

It seems that the "@" symbol is not vertically centered perfectly.

The vertical alignment is also inconsistent between the "@" symbol and "abc".